frontend/test/metabase/components/FieldValuesWidget.unit.spec.js

```diff
@@ -33,7 +33,7 @@ describe("FieldValuesWidget", () => {
 
       it("should have 'Enter some text' as the placeholder text", () => {
         renderFieldValuesWidget({ ...props });
-        screen.findByLabelText("Enter some text");
+        screen.getByPlaceholderText("Enter some text");
```

Comment:
A lot of these tests were not actually testing anything because `.findbyX` won't throw unless it is awaited. When I updated the assertions to work as intended, they were failing because they seemed to have not been updated in several years: e.g. labels were changed to placeholderText at some point.

Reply:
Good catch 👍